### DISCUSSION TITLE
Getting authentication error while working with pipeline code

### DISCUSSION THREAD

**Discussion by ghost** [community/community#77081] [New to GitHub]
Hi Team,

Getting authentication error while working with pipeline code, I have changed username and password, checked in windows credential manager, tested in command line by clonning git repo using PAT - all of them worked for me. My github is private repo. I am getting below error while running pipeline,
Cloning into 'repo'...
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/username/main.git/'
##[error]PowerShell exited with code '1'

**Comment by MantasTech** [comment]
Haaay,

Make sure your pipeline is using the right creds or PAT.
Confirm your PAT has access to private repos.
Double-check the repo URL in your pipeline code.
If you're using a CI/CD service, clear any old cached credentials.

Hope one of these fixes it!

**Comment by ghost** [comment]
i am using PAT authentication and checked repo access, and also url even then iam getting below error,
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/IC-ITG/de-ic.git/'
##[error]PowerShell exited with code '1'
